pom.xml

```diff
@@ -25,37 +25,31 @@
         <dependency>
```

Comment:
I think it would be a good idea to include the changes version in the CHANGES_NEXT_RELEASE file, in a style similar to the one used by https://github.com/telefonicaid/perseo-core/releases/tag/1.2.0:

```
Upgrade X library from Y to Z
```

```

Reply:
Fixed in 6cbfceaa3ae2dfea970f87e66a149e71d452008c

(Please, whenever you address a comment fixing someghing in the code, add a comment like the above, referencing the commit hash. That way, reviewers can trace how the comment has been addressed. Other possiblity is NTC to end a comment thread if, after discussion, result that nothings need to be done. You can have a look to the "PR protocol" at https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#pull-request-protocol)